src/partials/toc.hbs

```diff
@@ -11,6 +11,9 @@
                  </a>
               <a href="#" class="dislike-btn helpfull-btn" id="noBtn"  data-page-rating="dislike"> <i class="far fa-thumbs-down"></i> No</a>
```

Comment:
Editorial changes requested:
- " <h4> Is this page helpful ?</h4>" - remove space before the "?"
- "<a href="#" class="like-btn helpfull-btn" id="yesBtn" data-page-rating="like" >" - fix typo to "helpful"
- "<a href="#" class="dislike-btn helpfull-btn" id="noBtn"  data-page-rating="dislike">" - fix typo to "helpful"

Reply:
Okay, I"ll fix it.